packages/braintree-utils/src/utils/is-braintree-fastlane-profile-data.spec.ts

```diff
@@ -0,0 +1,11 @@
+import { getBraintreeFastlaneProfileDataMock } from '../';
+
+import isBraintreeFastlaneProfileData from './is-braintree-fastlane-profile-data';
+
+describe('isBraintreeFastlaneProfileData()', () => {
+    it('returns true if profile data belong to fastlane', () => {
```

Comment:
May I ask you to cover test where isBraintreeFastlaneProfileData returns false?

Reply:
Added